content/docs/ops/runbook/rotating-bosh.md

```diff
@@ -2,6 +2,7 @@
 menu:
   docs:
     parent: runbook
+layout: ops
 
```

Comment:
From this file through `content/docs/ops/runbook/troubleshooting-snort.md`, there's an extra linebreak between the `layout` line and the `title` line. It makes no functional difference, but I mention it for style consistency.